src/Command/CommandHelper.php

```diff
@@ -367,6 +366,11 @@ private static function getConfigFiles(
 			return [];
 		}
 
+		if (strpos('.php', $configFile) !== false) {
+			$neonAdapter = new PhpAdapter();
```

Comment:
I don't want to create a new object for each file, better to pass them both from the outside, like it's currently done with NeonAdapter.

Reply:
Also the variable name is wrong :) PhpAdapter should be in `$phpAdapter`.